cirras-underwriting-api/cirras-underwriting-service-api/src/main/java/ca/bc/gov/mal/cirras/underwriting/service/api/v1/impl/CirrasDataSyncServiceImpl.java

```diff
@@ -630,6 +639,117 @@ public void deletePolicy(Integer policyId,
 		logger.debug(">deletePolicy");
 
 	}
+
+	@Override
+	public Product getProduct(Integer productId, FactoryContext factoryContext, WebAdeAuthentication authentication)
+			throws ServiceException, NotFoundException, DaoException {
+
+		logger.debug("<getProduct");
+
+		Product result = null;
+
+		try {
+			ProductDto dto = productDao.fetch(productId);
+
+			if (dto != null) {
+				result = cirrasDataSyncFactory.getProduct(dto);
+			}
+			else {
+				throw new NotFoundException("Did not find product record: productId: " + productId);
```

Comment:
Other sync get services don't return a not found exception because it's only used for testing.